package.json

```diff
@@ -1,27 +1,28 @@
 {
-    "name": "vscode",
-    "version": "0.11.8",
-    "typings": "vscode.d.ts",
-    "scripts": {
-        "prepublish": "tsc"
-    },
-    "dependencies": {
-        "mocha": "^2.3.3",
-        "glob": "^5.0.15",
-        "request": "^2.67.0",
-        "semver": "^5.1.0",
-        "source-map-support": "^0.3.2",
-        "gulp-remote-src": "^0.4.0",
-        "gulp-vinyl-zip": "^1.1.2",
-        "gulp-symdest": "^1.0.0"
-    },
-    "license": "MIT",
-    "author": "Visual Studio Code Team",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/Microsoft/vscode-extension-vscode.git"
-    },
-    "bugs": {
-        "url": "https://github.com/Microsoft/vscode-extension-vscode/issues"
-    }
-}
\ No newline at end of file
+  "name": "vscode",
+  "version": "0.11.8",
+  "typings": "vscode.d.ts",
+  "scripts": {
+    "prepublish": "tsc"
+  },
+  "dependencies": {
+    "gh-got": "^2.4.0",
+    "glob": "^5.0.15",
+    "gulp-remote-src": "^0.4.0",
+    "gulp-symdest": "^1.0.0",
+    "gulp-vinyl-zip": "^1.1.2",
+    "mocha": "^2.3.3",
+    "request": "^2.67.0",
+    "semver": "^5.1.0",
+    "source-map-support": "^0.3.2"
+  },
+  "license": "MIT",
+  "author": "Visual Studio Code Team",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/vscode-extension-vscode.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/vscode-extension-vscode/issues"
+  }
+}
```

Comment:
It seems like `npm` automatically reformatted the entire file when I npm installed some extra dependencies.